src/data/fintype/basic.lean

```diff
@@ -1108,6 +1108,26 @@ Note this cannot be an instance as it needs `h`. -/
   (h : fintype.card β < fintype.card α) : is_empty (α ↪ β) :=
 ⟨λ f, let ⟨x, y, ne, feq⟩ := fintype.exists_ne_map_eq_of_card_lt f h in ne $ f.injective feq⟩
 
+/-- A constructive embedding of a fintype `α` in another fintype `β` when `card α ≤ card β`. -/
+def trunc_of_card_le {α β : Type*} [fintype α] [fintype β] [decidable_eq α] [decidable_eq β]
```

Comment:
I'm a bit worried the universes will infer incorrectly here. Can you use variables or explicit universes instead?

Reply:
There were already variables around. I have used them in this statement (and the neighboring ones).